source/manual/github-unavailable.html.md

```diff
@@ -6,62 +6,53 @@ layout: manual_layout
+We mirror all non-archived GitHub repositories tagged with `govuk` to AWS CodeCommit. This is achieved using a Mirror GitHub repositories workflow, which is [configured in terraform](https://github.com/alphagov/govuk-infrastructure/blob/main/terraform/deployments/github/mirror.tf) and carried out by this [github workflow](https://github.com/alphagov/govuk-infrastructure/blob/main/.github/workflows/mirror-repos.yml).
 
-The workaround is to disable Jenkins security, then configure the `Deploy_App` job to clone from AWS CodeCommit instead.
+### Checkout code in AWS CodeCommit
 
-## Disable Jenkins Security
+Since our deployment images are named via git tags, we'll need to checkout our code from AWS CodeCommit in order to make sure we're deploying the correct image and have an up-to-date copy of the code.
 
-If GitHub.com is down, we will not be able to log in to Jenkins.
-In this scenario, Jenkins security should be disabled to enable deployment:
-
-1. SSH to the Jenkins deploy instance: `gds govuk connect -e production ssh aws/jenkins`
-1. Disable Puppet: `govuk_puppet -r "Emergency Jenkins deploy" --disable`
-1. Edit the Jenkins configuration file: `sudo vim /var/lib/jenkins/config.xml`
-1. Replace `<useSecurity>true</useSecurity>` with `<useSecurity>false</useSecurity>` and save
-1. Restart Jenkins: `sudo service jenkins restart`
-
-Once security is disabled, anyone on GDS-trusted IPs will be able to invoke any Jenkins job, including `Deploy_App`. Do not leave security disabled for any longer than necessary.
-
-See the [Jenkins documentation](https://www.jenkins.io/doc/book/security/securing-jenkins/) for further details.
-
-## Deploy from AWS CodeCommit
-
-We mirror all non-archived GitHub repositories tagged with `govuk` to AWS CodeCommit every 2 hours. This is achieved using a ['Mirror GitHub repositories' Jenkins job](https://deploy.integration.publishing.service.gov.uk/job/Mirror_Github_Repositories/), which is [configured in govuk-puppet](https://github.com/alphagov/govuk-puppet/pull/11631/files) and the [govuk-repo-mirror repository](https://github.com/alphagov/govuk-repo-mirror).
-
-We can deploy from AWS CodeCommit instead of GitHub, if GitHub is down or if we need a private place to develop fixes for security vulnerabilities before they are deployed.
-
-To do so, trigger the [`Deploy_App`][Deploy_App] Jenkins job, but check the `DEPLOY_FROM_AWS_CODECOMMIT` box to deploy from AWS CodeCommit.
+Setup your local environment for [checking out code from CodeCommit](/manual/howto-checkout-and-commit-to-codecommit.html)
 
-### Making changes to code in AWS CodeCommit before deployment
+## Drill creating and deploying a branch from CodeCommit
 
-If you need to deploy from AWS CodeCommit because GitHub is down or you need to test a change of a sensitive nature, you will likely need to push a new branch of code to CodeCommit.
+If GitHub.com is down, old images can still be deployed manually in Argo.
```

Comment:
I think the "old" reference is a throwback from before you'd figured out how to create and deploy new images?

```suggestion
If GitHub.com is down, Docker images can still be created and then deployed manually in Argo.
```